File: hardway/04_Variable/01.py
# Set cars number equal 100
cars = 100
space_in_a_car = 4.0
drivers = 30
passegers = 90
cars_not_driven = 100 - 30
cars_driven = drivers
carpool_capacity = cars_driven * space_in_a_car
average_passergers_per_car = passegers / cars_driven

print "There are", cars, "cars available"
print "There are only", drivers, "drivers available"
print "There will be", cars_not_driven, "empty cars today"
print "We can transport", carpool_capacity, "people today"
print "We have", passegers, "to carpool today"
print "We need to put about", average_passergers_per_car, "in each car"

File: hardway/04_Variable/01.go
package main

import (
	"fmt"
)

func main() {
	cars := 90
	spaces_in_a_car := 4.0
	drivers := 30
	passengers := 90
	cars_not_driven := cars - passengers
	cars_driven := drivers
	carpool_capacity := cars * int(spaces_in_a_car)
	average_passergers_per_car := passengers / cars_driven

	fmt.Printf("There are %d cars available \n", cars)
	fmt.Printf("There are only %d drivers available \n", drivers)
	fmt.Printf("There will be %d empty cars to day \n", cars_not_driven)
	fmt.Printf("We can transport %d people today \n", carpool_capacity)
	fmt.Printf("We have %d to carpool today \n", passengers)
	fmt.Printf("We need to put about %d in each car \n", average_passergers_per_car)
}
